powertools-core/src/test/resources/unreadable.properties

```diff
@@ -0,0 +1 @@
+powertools.version=${project.version}
```

Comment:
Shouldn't this value be swapped with the `test.proprerties` empty value?

Reply:
The content is not really needed in this case, I can remove it if it is confusing. The file is [removed read access](https://github.com/eldimi/aws-lambda-powertools-java/blob/chore/user_agent_sdk_clients/powertools-core/src/test/java/software/amazon/lambda/powertools/core/internal/UserAgentConfigurerTest.java#L52) in a unit test to ensure version is set to "NA" in that case

Reply:
Ah interesting. It's a bit odd, because it looks that a valid input, but the name of the file does not match semantically, but I see now why it's called `unreadable`.

Reply:
I added [comments](https://github.com/eldimi/aws-lambda-powertools-java/blob/chore/user_agent_sdk_clients/powertools-core/src/test/resources/unreadable.properties) also for that one, since it was confusing